### PR TITLE
 chore: add volunteer Ugyenwangdi

### PR DESCRIPTION
**Pull request by Ugyenwangdi**

### CODE REVIEW

File: data/users/Ugyenwangdi.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Ugyen Wangdi",
+  "bio": "Full Stack developer | API developer ",
```
Comment (codewithvoid) [inline data/users/Ugyenwangdi.json:3]: you can just mention Full Stack Developer..that covers the latter

File: data/users/Ugyenwangdi.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "image": "https://avatars.githubusercontent.com/u/57824653?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["Django", "API", "JavaScript"]
```
Comment (codewithvoid) [inline data/users/Ugyenwangdi.json:8]: move javascript to separate entry `programming` like [here](https://github.com/codewithvoid/AskDevs/blob/main/data/users/codewithvoid.json)

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks @Ugyenwangdi for volunteering. Added few comments.

**Review by codewithvoid — APPROVED** [review]
Thanks @Ugyenwangdi for making the changes. Welcome!

**Comment by Ugyenwangdi** [comment]
> Thanks @Ugyenwangdi for making the changes. Welcome!

Thanks for helping out!

**Pull request merged**